network/alsp/manager/manager_test.go

```diff
@@ -1546,29 +1713,10 @@ func TestDisallowListNotification(t *testing.T) {
 	}, 1*time.Second, 10*time.Millisecond, "ALSP manager did not handle the misbehavior report")
 }
 
-// misbehaviorReportFixture creates a mock misbehavior report for a single origin id.
-// Args:
-// - t: the testing.T instance
-// - originID: the origin id of the misbehavior report
-// Returns:
-// - network.MisbehaviorReport: the misbehavior report
-// Note: the penalty of the misbehavior report is randomly chosen between -1 and -10.
-func misbehaviorReportFixture(t *testing.T, originID flow.Identifier) network.MisbehaviorReport {
-	return misbehaviorReportFixtureWithPenalty(t, originID, math.Min(-1, float64(-1-rand.Intn(10))))
-}
-
-func misbehaviorReportFixtureWithDefaultPenalty(t *testing.T, originID flow.Identifier) network.MisbehaviorReport {
-	return misbehaviorReportFixtureWithPenalty(t, originID, model.DefaultPenaltyValue)
-}
-
-func misbehaviorReportFixtureWithPenalty(t *testing.T, originID flow.Identifier, penalty float64) network.MisbehaviorReport {
-	report := mocknetwork.NewMisbehaviorReport(t)
-	report.On("OriginId").Return(originID)
-	report.On("Reason").Return(alsp.AllMisbehaviorTypes()[rand.Intn(len(alsp.AllMisbehaviorTypes()))])
-	report.On("Penalty").Return(penalty)
-
-	return report
-}
+////////////////////////////// TEST HELPERS ///////////////////////////////////////////////////////////////////////////////
+// The following functions are helpers for the tests. It wasn't feasible to put them in a helper file in the alspmgr_test
+// package because that would break encapsulation of the ALSP manager and require making some fields exportable.
+// Putting them in alspmgr package would cause a circular import cycle. Therefore, they are put in the internal test package here.
```

Comment:
Maybe moving these test helpers into a separate file but in the same package is more maintainable, e.g., `testHelpers.go`.

Reply:
Unfortunately, Go doesn't allow that. Only `*_test.go` files are allowed to be in the `alspmgr_test` package. If I try and create a new `test_helpers.go` file with the same `alspmgr_test` package, the compiler complains: `Multiple packages in the directory: alspmgr, alspmgr_test`.